scripts/tmux-extrakto.sh

```diff
@@ -36,10 +36,18 @@ if [[ "$open_tool" == "auto" ]]; then
   esac
 fi
 
+if [[ -z $EDITOR ]]; then
+  # fallback
+  editor="vi"
+else
+  editor="$EDITOR"
+fi
+
 function capture() {
 
   header="tab=insert, enter=copy"
   if [ -n "$open_tool" ]; then header="$header, ctrl-o=open"; fi
+  header="$header, ctrl-e=open in $editor"
```

Comment:
I thik `ctrl-e=edit` should be sufficient (also for example my `$EDITOR` command includes parameters and this looks a bit strange)